dev/io.openliberty.jakarta.concurrency.3.0_fat_tck/publish/servers/ConcurrentTCKFullServer/server.xml

```diff
@@ -11,15 +11,7 @@
 <server>
     <featureManager>
         <!-- Features being tested -->
-        <!--TODO once all of EE 10 is beta, could enable <feature>jakartaee-10.0</feature> or leave the individual features -->
-        <feature>concurrent-3.0</feature>
-        <feature>jdbc-4.2</feature>
-        <feature>enterpriseBeans-4.0</feature>
-        <feature>cdi-4.0</feature>
-        <feature>pages-3.1</feature>
-        <!-- Supporting features -->
-        <feature>jndi-1.0</feature>
-        <feature>appSecurity-5.0</feature>
+		<feature>jakartaee-10.0</feature>
```

Comment:
I could see the list of individual features that the TCK requires being useful for a future Jakarta release where only some but not all of the EE vNext features are available and no platform feature exists yet.  In that case, it would be useful to selectively choose the features.  Maybe have the list of required features commented out for future reference?